bin/utils.py

```diff
@@ -16,14 +16,25 @@ def data_prep_from_saved_model(
     # Extract graph definition
     gdef = sess.graph_def
 
+    # Trim graph to keep only the nodes related to data pre-processing
+    # tf.graph_util.extract_sub_graph will be removed in future tf version
+    data_prep_end_node_name = data_prep_end_node.split(":")[0]
+
+    try:
```

Comment:
I suggest moving this try/except up into the import statements.  you can import the function directly instead of using the higher-level module syntax, which will keep the core code cleaner

Reply:
thanks for the suggestion, let me fix that.